test/scss/feature-targeting.scss

```diff
@@ -4,35 +4,54 @@
 @import "@material/feature-targeting/functions";
 @import "@material/menu/mixins";
 @import "@material/radio/mixins";
+@import "@material/ripple/mixins";
 @import "@material/typography/mixins";
 
 // Button
-
-@include mdc-button($query: mdc-feature-any());
+.mdc-button {
```

Comment:
I know we talked about this elsewhere, but looking at this, it occurs to me that it's incorrect (and thus misleading if anyone were to take this as reference, although they shouldn't) to invoke a component's main `mdc-foo` mixin within a selector. (This only goes for components, e.g. not elevation/typography)

If we're doing this per-component, perhaps we should reserve that treatment only for individual public mixins, not the main one capable of emitting all the base styles (which would emit this selector itself)?

Alternatively we could just have a `.test` selector around the entire set...

Reply:
yeah that's a good point, I'll just put `.test` around everything

Reply:
went with `.mdc-test` in the end due to the linter